source/jsonnet/england-wales/individual/blocks/identity-and-health/ethnic_group_black.jsonnet

```diff
@@ -76,24 +65,53 @@ local proxyWalesTitle = {
   ],
 };
 
-function(region_code) {
-  type: 'Question',
-  id: 'black-ethnic-group',
-  question_variants: [
-    {
-      question: question(nonProxyEnglandTitle, nonProxyWalesTitle, region_code, nonProxyDefinitionDescription),
-      when: [rules.isNotProxy],
-    },
-    {
-      question: question(proxyEnglandTitle, proxyWalesTitle, region_code, proxyDefinitionDescription),
-      when: [rules.isProxy],
-    },
-  ],
-  routing_rules: [
-    {
-      goto: {
-        block: 'religion',
+
+function(region_code) (
+  local optionLabelValue = if region_code == 'GB-WLS' then 'Any other Black, Black Welsh Black British or Caribbean background'
```

Comment:
Missing comma.

Where is this string sourced?. I can't find it in the google sheet or the attached overflow?

Reply:
Its the piece talked about a few days ago, when Andy said just do it. Its related to the comment Rhys made above